build/RuntimeStore.targets

```diff
@@ -0,0 +1,211 @@
+<Project>
+  <Import Project="common.props" />
+  <Import Project="existingStores.props" />
```

Comment:
Note, this will no longer be needed after moving to manifest based delta zip calculations.